renderdoc/android/android.cpp

```diff
@@ -993,6 +993,9 @@ struct AndroidController : public IDeviceProtocolHandler
       // 32-bit remote server.
       Android::adbExecCommand(deviceID, "shell am start -n " + GetRenderDocPackageForABI(abis.back()) +
                                             "/.Loader -e renderdoccmd remoteserver");
+
+      // enable profiling with gatord to measure hardware counters
+      Android::adbExecCommand(deviceID, "shell setprop security.perf_harden 0");
```

Comment:
Is there any potential problem with setting this unconditionally? E.g. will this cause problems on non-ARM devices or on versions of android where this property doesn't exist?

Reply:
We have tested this on a device which does not have this property there were no problems (with an older Android and also on a non-ARM GPU Android). 

On the other side we have moved this "setprop" call to be done when the capture is opened and when the capture file is closed the property will be set back to "1".

I'll update the PR today.